R/plot_fit.R

```diff
@@ -108,30 +108,31 @@ spim_plot_fit_traces <- function(samples, region = NULL) {
 
   if (is.null(region)) {
 
-    n_pars <- length(colnames(samples$pars))
+    n_pars <- length(colnames(samples$pars_full))
 
     par(mfrow = rep(ceiling(sqrt(n_pars + 2)), 2),
         mar = c(3, 3, 2, 1),
         mgp = c(2, 0.5, 0),
         oma = c(1, 1, 1, 1))
 
-    for (nm in colnames(samples$pars)) {
-      plot_traces1(samples$pars[, nm], nm)
+    for (nm in colnames(samples$pars_full)) {
```

Comment:
Do you think it's worth worth including any visual clue as to the real sampling frequency here? something like `rug()` with the index of the sampled cases? that would visually show the burnin and the degree to which autocorrelation is biting us.  I think you want `rug(samples$iteration[samples$chain == 1])`

Reply:
Added